package.json

```diff
@@ -177,7 +178,7 @@
     "terriajs-tiff-imagery-provider": "2.13.3",
     "thredds-catalog-crawler": "0.0.7",
     "ts-essentials": "^5.0.0",
-    "typescript": "~5.2.0",
+    "typescript": "^5.7.2",
```

Comment:
TypeScript 5.6 introduces `--noUncheckedSideEffectImports` (https://devblogs.microsoft.com/typescript/announcing-typescript-5-6/#the---nouncheckedsideeffectimports-option).

I was hoping that would have found the casing difference between import/export in #7344, but it didn't. It still seems like a nice thing to check though, can that option be enabled in `tsconfig.json`? I'm not sure where the ambient module declaration should go, but adding:
```
declare module "*.css" {}
```
in `lib/ThirdParty/globals.d.ts` makes tsc with `noUncheckedSideEffectImports` enabled pass for me.